pkg/cvo/availableupdates.go

```diff
@@ -127,7 +128,7 @@ func calculateAvailableUpdatesStatus(clusterID, upstream, channel, version strin
 		}
 	}
 
-	updates, err := checkForUpdate(clusterID, upstream, channel, currentVersion)
```

Comment:
I would assume this is the only place that needs the ctx, why do any body up the chain care about ctx for checking updates?

Reply:
I can cut the chain at any arbitrary point that you prefer, but please do note that the topmost `Until` layer already takes a stop channel, but doesn't propagate that down. I'd rather suggest to leave the TODO here and later fix the caller to unify the cancellation chain.

Reply:
FYI I landed a fix in k8s master to propagate the context through `Until`: https://github.com/kubernetes/kubernetes/pull/72928